test/functional/eval/interrupt_spec.lua

```diff
@@ -0,0 +1,61 @@
+local helpers = require('test.functional.helpers')(after_each)
+
+local command = helpers.command
+local meths = helpers.meths
+local clear = helpers.clear
+local sleep = helpers.sleep
+local wait = helpers.wait
+local feed = helpers.feed
+local eq = helpers.eq
+
+local dur
+local min_dur = 8
+local len = 131072
+
+describe('List support code', function()
+  if not pending('does not actually allows interrupting with just got_int', function() end) then return end
```

Comment:
The `if not pending` check seems like a mistake. Would be clearer:

```
pending('does not actually allows interrupting with just got_int', function() end)
return
```

Or remove it totally?

Reply:
I wrote it this way so that it is clear that return is there due to pending and to write everything in one line (one-line if is sometimes used, two syntactically unconnected statements on one line are not).